tests/test_types.py

```diff
@@ -32,6 +33,14 @@ def test_is_union_with_non_union():
     assert not is_union(int)
 
 
+def test_is_tuple_with_tuple():
+    assert is_tuple(Tuple[int, float, str])
```

Comment:
I've added tests for first three cases. Last two cases are not valid, since you cannot use brackets with type objects in type hints.

Reply:
@michalchrobok That's not true. Since Python 3.9 you can, see [PEP 585](https://peps.python.org/pep-0585/). You can see an example in the codebase here of how to skip tests on versions below a threshold:

- https://github.com/konradhalas/dacite/blob/master/tests/test_types.py#L36
- https://github.com/konradhalas/dacite/blob/master/tests/common.py#L5

Reply:
I opted for naming the decorator after the PEP [here](https://github.com/konradhalas/dacite/pull/184/files#diff-53eb316ef7b18ba47acdd955cdaa7ca5467cd5ba6f5cf8ed92c68a421902d120R6), but I don't know yet what @konradhalas thinks about that.

Reply:
@antonagestam Sorry, I wasn't familiar with this feature of Python 3.9. I have added missing tests in [fceef0b](https://github.com/konradhalas/dacite/pull/175/commits/fceef0bd11ecf29db41fd1632dd74f6c38c4ab37). I've opted for decorator name more similar to existing ones.